GTBitsOfGood/gt-scheduler-website
# Add filter for start and end times

## Fix the bug where 'my classes' won't show up on 2nd col of Course Search tab

Issue Number(s): #10 

What does this PR change and why?
If you have courses added in `Schedule` tab and try to look up for that course in `Course Schedule` tab, you will not see it. This PR fixes that bug so that courses show up on the 2nd column of `Course Search` regardless of whether they are already added by the user.

It also works with Edmund's work #13 . Clicking on `i` button in the 2nd column will show data on 3rd column whoo

### Checklist


### Critical Changes

- Made significant changes to `Course/index.tsx` related to how it colors the course cards

### Related PRs

- #15, #16 

### How to Test

Add courses from `Schedule` tab, go to `Course Schedule` tab, search for the added course, check they show up, click the `i` button, check that the third column (the table) is populated properly (this part is not fully tested).
